src/wast-lexer.cc

```diff
@@ -354,6 +354,37 @@ Token WastLexer::GetStringToken(WastParser* parser) {
             }
             break;
 
+          case 'u': {
+            token_start_ = cursor_ - 2;
+            if (ReadChar() != '{') {
+              goto error;
```

Comment:
Currently every error displays the same simple error message. If you prefer, I can add more sophisticated error messages as well.

Reply:
I think this is fine for now.